test/integration/policy_info_metric_test.go

```diff
@@ -0,0 +1,185 @@
+		By("Creating a policy")
+		oc("apply", "-f", compliantPolicyYaml, "-n", userNamespace, "--kubeconfig="+kubeconfigHub)
+		// Don't need to check compliance - just need to guarantee there is a policy in the cluster
+
+		token, err := oc("whoami", "-t")
+		Expect(err).To(BeNil())
+		Eventually(func() interface{} {
+			resp, _, err := getMetricsFromRoute(strings.TrimSpace(token))
+			if err != nil {
+				return err
+			}
+			return resp
+		}, defaultTimeoutSeconds, 1).Should(ContainSubstring("HELP " + metricName))
+	})
+	It("Checks that a compliant policy reports a metric of 0", func() {
+		By("Creating a compliant policy")
+		oc("apply", "-f", compliantPolicyYaml, "-n", userNamespace, "--kubeconfig="+kubeconfigHub)
+		Eventually(
+			getComplianceState(compliantPolicyName),
+			defaultTimeoutSeconds,
+			1,
+		).Should(Equal(policiesv1.Compliant))
+
+		By("Checking the policy metric")
+		token, err := oc("whoami", "-t")
+		Expect(err).To(BeNil())
+		policyLabel := `policy="` + compliantPolicyName + `"`
+		Eventually(func() interface{} {
+			resp, _, err := getMetricsFromRoute(strings.TrimSpace(token))
+			if err != nil {
+				return err
+			}
+			return resp
+		}, defaultTimeoutSeconds, 1).Should(matchMetricValue(metricName, policyLabel, "0"))
+	})
+	It("Checks that a noncompliant policy reports a metric of 1", func() {
+		By("Creating a noncompliant policy")
+		oc("apply", "-f", noncompliantPolicyYaml, "-n", userNamespace, "--kubeconfig="+kubeconfigHub)
+		Eventually(
+			getComplianceState(noncompliantPolicyName),
+			defaultTimeoutSeconds,
+			1,
+		).Should(Equal(policiesv1.NonCompliant))
+
+		By("Checking the policy metric")
+		token, err := oc("whoami", "-t")
+		Expect(err).To(BeNil())
+		policyLabel := `policy="` + noncompliantPolicyName + `"`
+		Eventually(func() interface{} {
+			resp, _, err := getMetricsFromRoute(strings.TrimSpace(token))
+			if err != nil {
+				return err
+			}
+			return resp
+		}, defaultTimeoutSeconds, 1).Should(matchMetricValue(metricName, policyLabel, "1"))
+	})
+	It("Cleans up", func() {
+		oc("delete", "-f", compliantPolicyYaml, "-n", userNamespace, "--kubeconfig="+kubeconfigHub)
+		oc("delete", "-f", noncompliantPolicyYaml, "-n", userNamespace, "--kubeconfig="+kubeconfigHub)
+		oc("delete", "route", "-n", ocmNS, "-l", propagatorMetricsSelector)
+	})
+})
+
+func oc(args ...string) (string, error) {
+	output, err := exec.Command("oc", args...).CombinedOutput()
+	if len(args) > 0 && args[0] != "whoami" {
+		fmt.Println(string(output))
+	}
+	return string(output), err
+}
+
+func getMetricsFromRoute(authToken string) (body, status string, err error) {
+	client := &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
+			},
+		},
+		Timeout: 5 * time.Second,
+	}
+	req, err := http.NewRequest("GET", "https://"+routeURL+"/metrics", nil)
+	if err != nil {
+		return "", "", err
+	}
+	if len(authToken) > 0 {
```

Comment:
Optional: I find this more intuitive to read.
```suggestion
	if authToken != "" {
```